jekyll/_cci2/workflows.md

```diff
@@ -216,6 +216,71 @@ workflows:
 
 In the example, `filters` is set with the `branches` key and the `only` key with the branch name. Any branches that match the value of `only` will run the job. Branches matching the value of `ignore` will not run the job. See the [Sample Sequential Workflow config with Branching](https://github.com/CircleCI-Public/circleci-demo-workflows/blob/sequential-branch-filter/.circleci/config.yml) for a full example.
 
+## Git Tag Job Execution
+Git tags must be declared in each job and in the workflow. CircleCI will not build Git tags unless they are specified under each job key. The following example shows a workflow with jobs filtered to ignore tags starting with `testing-`. 
```

Comment:
We can remove this: `The following example shows a workflow with jobs filtered to ignore tags starting with `testing-`. since it is repeated.